cmd/web-frontend/main.go

```diff
@@ -0,0 +1,20 @@
+package main
+
+import (
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	"github.com/ucl-arc-tre/portal/internal/graceful"
+	"github.com/ucl-arc-tre/portal/internal/router"
+)
+
+func main() {
+	router := router.New()
+	router.LoadHTMLFiles("./index.html")
```

Comment:
does this work because go knows to look for the web folder?

Reply:
How can we test if this works? I can see that we can access the development server on port 8000, but how can we access this release build to test that the static assets are being served correctly?

Reply:
going forward this should be cypress, but for now you should be able to 

```bash
docker compose up nginx --build
```

Reply:
Is this covered in the docs somewhere? It was straightforward to set everything up, but a bit more descriptive context around the project structure and the different components involved would be good I think. 

Also happy for that to go in a follow up PR though if we wanted this to be kept specifically to the scaffold to get going with things.

Reply:
> Is this covered in the docs somewhere?

I don't think so

> a bit more descriptive context around the project structure and the different components involved would be good I think

sure, if that'd be helpful for new people – I vote in a followup PR